src/11_args.py

```diff
@@ -5,13 +5,20 @@
 # the sum. This is what you'd consider to be a regular, normal function.
 
 # YOUR CODE HERE
+def f1(num1, num2):
+    return num1 + num2
 
 print(f1(1, 2))
 
 # Write a function f2 that takes any number of integer arguments and prints the
 # sum. Google for "python arbitrary arguments" and look for "*args"
 
 # YOUR CODE HERE
+def f2(*args):
+    total = 0
+    for num in args:
+        total += num
+    return total
```

Comment:
Another fun way you could write this is using the sum() method in Python like this: 
```
def f2(*args):
    return sum(args)
```